rfc/rfc-0000-common-blueprint.md

```diff
@@ -0,0 +1,220 @@
+# Meta
+[meta]: #meta
+- Name: Common Blueprint Architecture
+- Start Date: 2022-03-29
+- Author(s): @emmjohnson @martyspiewak
+- Status: Draft
+- RFC Pull Request: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This RFC proposes collapsing all current "Blueprints" and templates (not including `ClusterRunTemplate`) into a common type, `ClusterBlueprint`. 
```

Comment:
maybe worth calling out **why** not clusterruntemplate? I imagine for most readers not super familiar with it, this might look confusing

perhaps something in the lines of

```suggestion
This RFC proposes collapsing all current "Blueprints" and templates (not including `ClusterRunTemplate`, as that resource is in the domain of [Runnable](https://cartographer.sh/docs/v0.3.0/reference/runnable/)) into a common type, `ClusterBlueprint`. 
```

?